### PR TITLE
A few things

### PR DESCRIPTION
**Pull request by solsticedhiver**
Tell me what you think about this.

This goes from fixing typo, to check the architecture in the Makefile, install polkit.rules, and the last ...

I don't know yet. Is the patch really needed ? Waiting comment in raspberrypi os forum, if any comes in this [thread](https://forums.raspberrypi.com/viewtopic.php?p=1960785&hilit=kodi+systemd#p1960785)

### CODE REVIEW

**Review by graysky2 — DISMISSED** [review]

**Comment by graysky2** [comment]
Looks decent.  Let's see what the RPiOS folks say. @popcornmix - can you comment on the modifications made and on using the service on RPiOS in general?

See [the forum post](https://forums.raspberrypi.com/viewtopic.php?p=1960785) and https://github.com/graysky2/kodi-standalone-service/issues/37

File: Makefile [hunk 1]
```diff
@@ -14,34 +14,48 @@ INSTALL_DIR = $(INSTALL) -d
 INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
+ifneq ($(cat /proc/cpuinfo | grep -q ARM),"")
```
Comment (graysky2) [inline Makefile]: I am unfamiliar with using an error code like this, but in testing, this check always selects x86 even on an ARM device.

An aside: `grep -q ARM /proc/cpuinfo` would be better.

**Review by graysky2 — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -14,34 +14,53 @@ INSTALL_DIR = $(INSTALL) -d
 INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
+IS_ARCH_ARM := $(grep -q ARM /proc/cpuinfo; echo $$?)
+ifeq ($(IS_ARCH_ARM), 0)
```
Comment (graysky2) [inline Makefile:19]: I still think this is not working.  The variable IS_ARCH_ARM does not get assigned 0 or 1 it seems to be a null so the check always ends up as x86.

EDIT: I think you need: `IS_ARCH_ARM := $(shell grep -q ARM /proc/cpuinfo; echo $$?)`

**Review by graysky2 — CHANGES_REQUESTED** [review]
Still not working

File: Makefile [hunk 3]
```diff
@@ -28,7 +29,9 @@ common/$(PN):
 install-common:
 	$(INSTALL_DIR) "$(DESTDIR)$(UDEVDIR)"
 	$(INSTALL_DIR) "$(DESTDIR)$(ENVDIR)"
+	$(INSTALL_DIR) "$(DESTDIR)$(POLKDIR)"
```
Comment (graysky2) [inline Makefile:32]: The polkit dir and file is only needed for arm not x86_64.  Please move these two lines under that conditional.

Comment (graysky2) [inline Makefile:32]: This is only needed for ARM; please move under line 37

File: Makefile [hunk 4]
```diff
@@ -28,7 +29,9 @@ common/$(PN):
...
 	$(INSTALL_DATA) $(ARCH)/udev/99-kodi.rules "$(DESTDIR)$(UDEVDIR)/99-kodi.rules"
+	$(INSTALL_DATA) $(ARCH)/polkit/polkit.rules "$(DESTDIR)$(POLKDIR)/99-kodi.rules"
```
Comment (graysky2) [inline Makefile:34]: This is only needed for ARM; please move under line 37

**Review by graysky2 — CHANGES_REQUESTED** [review]
polkit dir and file only needed for ARM, please move

**Comment by graysky2** [comment]
I corrected it on the shell.  Not very adept with github's interface.  Let's wait to see about updating the README based on comment.  Closing this since I cherry-picked the changes.  Thank you for contributing!

**Comment by solsticedhiver** [comment]
No problem. I saw the problem about polkit dir and arm,a nd push and squash a new commit. 
Before I had to go. sorry.

And that's weird, it was working for me without the "shell" keyword. (But yes, I wrongly copied that without the correct shell keyword /o\)
Never mind.

**Comment by graysky2** [comment]
It's working fine now in my testing.  Good improvement.

**Comment by ghen2** [comment]
> polkit dir and file only needed for ARM, please move

Does this mean the x86_64 package can drop the polkit dependency ?
(kodi-standalone-service is the only dependent on polkit on my Arch HTPC)

**Comment by ghen2** [comment]
No, it seems needed for the poweroff/reboot capabilities (even without the polkit rules file).

**Pull request closed without merging**